docs/source/methods/depletion.rst

```diff
@@ -257,3 +257,94 @@ choose one of two methods for estimating the heating rate, including:
 
 The method for normalization can be chosen through the ``normalization_mode``
 argument to the :class:`openmc.deplete.CoupledOperator` class.
+
+--------------
+Transfer Rates
+--------------
+
+OpenMC allows continuous removal or feed of isotopes by adding an
+extra transfer rate term to the depletion matrix. An application of this feature
+is the chemical processing of Molten Salt Reactors (MSRs), and can be used to
+model fission products removal or feed fresh fuel into the system.
+
+A transfer rate as defined here is the rate at which nuclides are
+continuously removed/feed from/to a material.
+
+.. note::
+
+    A transfer rate can be positive or negative, indicating removal or feed
+    respectively.
+
+Mathematically, it can be thought as an additional proportional
+term :math:`\mathbf{T}` to the depletion equation, which can be written as:
```

Comment:
```suggestion
Mathematically, it can be thought as an additional term :math:`\mathbf{T}`
in the depletion equation that is proportional to the nuclide density, which can be written as:
```